asm/istio/expansion/gen-eastwest-gateway.sh

```diff
@@ -92,15 +90,12 @@ IOP=$(cat <<EOF
 $IOP
         enabled: true
         k8s:
-          env:
-            # sni-dnat adds the clusters required for AUTO_PASSTHROUGH mode
-            - name: ISTIO_META_ROUTER_MODE
-              value: "sni-dnat"
```

Comment:
Hi @howardjohn, is this change expected?

Reply:
this setting hasn't been needed for a few versions, so as long as this isn't used with super old versions then yes.